doc/source/new/virtualtables.rst

```diff
@@ -0,0 +1,586 @@
+                                        repair_session_space_in_mb |                                                                    62
+                repaired_data_tracking_for_partition_reads_enabled |                                                                 false
+                    repaired_data_tracking_for_range_reads_enabled |                                                                 false
+                       report_unconfirmed_repaired_data_mismatches |                                                                 false
+                                             request_timeout_in_ms |                                                                 10000
+                                                      role_manager |                                                  CassandraRoleManager
+                                                       rpc_address |                                                             localhost
+                                                     rpc_interface |                                                                  null
+                                            saved_caches_directory |                            /home/ec2-user/cassandra/data/saved_caches
+                                                     seed_provider | org.apache.cassandra.locator.SimpleSeedProvider{seeds=127.0.0.1:7000}
+                               server_encryption_options_algorithm |                                                                  null
+                                server_encryption_options_protocol |                                                                   TLS
+                                      slow_query_log_timeout_in_ms |                                                                   500
+                                                  ssl_storage_port |                                                                  7001
+                                            start_native_transport |                                                                  true
+                                                      storage_port |                                                                  7000
+                                            stream_entire_sstables |                                                                  true
+               transparent_data_encryption_options_chunk_length_kb |                                                                    64
+                        transparent_data_encryption_options_cipher |                                                  AES/CBC/PKCS5Padding
+                       transparent_data_encryption_options_enabled |                                                                 false
+                     transparent_data_encryption_options_iv_length |                                                                    
+ 16
+ (224 rows)

```

Comment:
Removed several listing settings. Added `...`

Reply:
i was thinking more along the lines of only listing like 10ish settings to give idea of output so its not this huge scroll. We dont really need an exhaustive list of all settings since that will be very hard to keep correct and is just a big data dump.

Reply:
Removed scroll. Only the basic settings are listed such as cluster name and storage port.